### DISCUSSION TITLE
v1.3.5 - New Additions and Important Fixes therminal runaway

### DISCUSSION THREAD

**Discussion by ghost** [Jyers/Marlin#1270] [General Discussion]
Hi,
when i use this firmware i get a therminal runaway message 
when i use the stock firmware everything is normal no problems at all
even when i do the auto pid tune on the 1.3.5 firmware the problem still remains  
does anyone has the same problem or how to fix this

**Comment by Cal-LF** [comment]
@Jyers There are also reports of this on FB group. More change of thermal settings may be needed.

**Reply by ghost** [reply]
i dont have fb sorry for that 
When is this going to be solved ? or is it possible that i look into this 
im getting the feeling that to much is enabled for the board which could cause problems 
Correct me if im wrong

**Reply by Cal-LF** [reply]
@pbajai If you compile your own, the thermal safety settings can be changed. Text file shows the settings to change.
[Thermal Protection.txt](https://github.com/Jyers/Marlin/files/6950990/Thermal.Protection.txt)

**Reply by ghost** [reply]
thanks, but i like the menu for the live level how to put this in ?
i have tried to comply it like this but never got so far
